packages/ripple-nuxt-tide/lib/components/AppForm.vue

```diff
@@ -53,7 +53,7 @@ export default {
     checkFieldStates () {
       this.fields.forEach(field => {
         if (field.states) {
-          conditionalLogic(field, field.states, this.formData)
+          conditionalLogic(field, this.formData)
```

Comment:
@tim-yao  This is a breaking change, if any other sites are using conditional logic.

Reply:
@alan-cole only App-form is using it as I can see inside our org projects, so it should be fine.